language/stdlib/modules/event.mvir

```diff
@@ -78,10 +78,10 @@ module Event {
 
     // Native procedure that writes to the actual event stream in Event store
     // This will replace the "native" portion of EmitEvent bytecode
-    native write_to_event_store<T>(guid: bytearray, count: u64, msg: T);
+    native write_to_event_store<T: unrestricted>(guid: bytearray, count: u64, msg: T);
```

Comment:
Could you add a comment emphasizing the importance of `unrestricted` here (as I mentioned previously)?